index.bs

```diff
@@ -931,6 +933,50 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+### Advertise Platform Authenticator Support - isPlatformAuthenticatorReady() method ### {#isPlatformAuthenticatorReady}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorReady">
+This method can be used by [=[RPS]=] to inform the user that they can create a new credential with one or more platform authenticators.
+The method gives [=[RPS]=] the ability to inform the client that they support registering credentials with [=platform authenticators=]
+while giving users control over whether this information is appropriate to provide to the RP.
+When the method is invoked, the [=client=] will search for the existence of a [=platform authenticator=]
+capable of registering credentials.
+Upon finding one, the [=client=] will determine whether the user would be interested in
+creating a credential based on a number of factors, which may include the user's explicitly stated intentions.
+If such interest is found, the promise will be resolved with the value of <i>True</i>.
+Based on the result, the [=[RP]=] can take further actions to guide the user to create a credential.
+
```

Comment:
> Whether the user is running in private or incognito mode.

I would be very sad indeed if this method was useful to detect incognito mode. We're trying to remove those vectors, not to add new ones. Would y'all mind adding this to the privacy considerations section?